src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+export const CATEGORY_ACTION = {
+  STATUS: 'category_status',
+};
+
+const initialState = {
+  categories: [],
+};
+
+export default function categoryReducer(state = initialState, action) {
+  switch (action.type) {
+    case CATEGORY_ACTION.STATUS:
+      return 'Under construction';
+    default:
+      return state;
+  }
+}
```

Comment:
- [ ] [Optional] You using the old way to create a redux reducer in the `redux toolkit` you suppose to use `createSlice` to [create a Redux State Slice](https://redux-toolkit.js.org/tutorials/quick-start#create-a-redux-state-slice).